api/inventorypb/agents.proto

```diff
@@ -964,6 +1039,20 @@ service Agents {
       body: "*"
     };
   }
+  // AddQANPostgreSQLPgStatementsAgent adds PostgreSQL Pg Stat Statements QAN Agent.
+  rpc AddQANPostgreSQLPgStatementsAgent(AddQANPostgreSQLPgStatementsAgentRequest) returns (AddQANPostgreSQLPgStatementsAgentResponse) {
+    option (google.api.http) = {
+      post: "/v0/inventory/Agents/AddQANPostgreSQLPgStatementsAgent"
+      body: "*"
+    };
+  }
+  // ChangeQANPostgreSQLPgStatementsAgent changes PostgreSQL Pg Stat Statements QAN Agent.
+  rpc ChangeQANPostgreSQLPgStatementsAgent(ChangeQANPostgreSQLPgStatementsAgentRequest) returns (ChangeQANPostgreSQLPgStatementsAgentResponse) {
```

Comment:
Please make PR to pmm-managed with stubs for those methods so we could update without complication failures.

Reply:
ok, please, don't merge until I make PR :)